.ci/lint

```diff
@@ -47,7 +47,9 @@ pipeline {
                     steps {
                         sh '''
                         eval "$(pyenv init - --no-rehash)"
-                        egrep -i '^salt/.*\\.py$|^setup\\.py$' file-list-changed.log | xargs -r '--delimiter=\\n' tox -e pylint-salt | tee pylint-report-salt.log
+                        egrep -Ei '^salt/.*\\.py$|^setup\\.py$' file-list-changed.log | xargs -r '--delimiter=\\n' tox -e pylint-salt | tee pylint-report-salt.log
+                        # remove color escape coding
+                        sed -ri 's/\x1B\[([0-9]{1,2}(;[0-9]{1,2})?)?[m|K]//g' pylint-report-salt.log
```

Comment:
okay explain to me what this is actually needed for? the colors are automatically removed when pylint is piped to something, unless something is getting through.

Reply:
Save the current xml version and you will see them.   I suspect PY_COLOR forces it.   And is done so, so the console out put looks nice in Jenkins